index.js

```diff
@@ -611,7 +617,7 @@ class Plugin extends MiniAccountsPlugin {
     // if the claim we're signing is for more than the channel's max balance
     // minus half the minimum balance, add some funds
     if (!account.isFunding() && aboveThreshold) {
-      debug('adding funds to channel. account=', account.getAccount())
+      this._log.trace('adding funds to channel. account=', account.getAccount())
```

Comment:
The fund operations spends some of our XRP so it might be good to have it on the info level. If there's a ton of these going off then it suggests something is wrong